src/test/java/com/aws/greengrass/shadowmanager/ipc/UpdateThingShadowRequestHandlerTest.java

```diff
@@ -735,6 +738,22 @@ void GIVEN_bad_update_document_with_no_state_node_WHEN_handle_request_THEN_throw
         assertInvalidArgumentsErrorFromPayloadUpdate(null, badUpdateRequest, expectedErrorString, expectedErrorCode, context);
     }
 
+    @Test
+    void GIVEN_update_document_with_delta_WHEN_handle_request_THEN_delta_is_dropped_from_data() throws IOException, URISyntaxException {
+        byte[] badUpdateRequest =
```

Comment:
We should capture the message sent to Synchandler and make sure that even that object does not have the `delta` node.

Reply:
This test covers it, without my change the test fails.

Reply:
What this is testing is what the method is returning. I just want to completely make it future proof if we decide to use a different object we send to sync.

Reply:
Updated.